cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
Delete for now as its' not being used?

Reply:
Sounds good!

Reply:
actually, we can't delete this file without it breaking cypress because it expects to find it to see if there are commands.

Reply:
Oh ok. we can keep it then.